src/vsr/client_replies.zig

```diff
@@ -308,7 +308,7 @@ pub fn ClientRepliesType(comptime Storage: type) type {
             maybe(client_replies.writing.isSet(slot.index));
             assert(message.header.command == .reply);
             // There is never any need to write a body-less message, since the header is
-            // stored safely in the client sessions superblock trailer.
+            // stored safely in the superblock header's `client_sessions`.
```

Comment:
This makes it sound like sessions are stored in the superblock header, rather than in a checkpoint
trailer.